python/py-fann2/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        FutureLinkCorporation fann2 1.2.0
+name                py-fann2
+revision            0
+checksums           rmd160  751e4b4ea2104bbd6500c601bd153e50a2709b27 \
+                    sha256  63b2a6dde0b1d7f9a642598b86f223d0880cb87aef09024e19cc829840c1e3f0 \
+                    size    65126
+
+categories-append   science
+platforms           darwin
+supported_archs     noarch
+license             LGPL-2
+maintainers         {fusion.gat.com:smithsp @smithsp} openmaintainer
+description         Python bindings for Fast Artificial Neural Networks 2.2.0 (FANN >= 2.2.0).
+long_description    This is a python binding for Fast Artificial Neural Network Library (FANN >= 2.2.0) that \
+    implements multi-layer artificial neural networks with support for both fully-connected and sparsely-connected \
+    networks. It includes a framework for easily handling training data sets. It is easy to use, versatile, well-\ documented, and fast.
+
+python.versions     27 35 36 37 38
+
+github.tarball_from archive
+
+build.env-append PREFIX=$prefix
+destroot.env-append PREFIX=$prefix
+
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
+    livecheck.type       none
+    depends_build-append port:fann
+    depends_build-append port:swig
```

```suggestion
    depends_build-append port:py${python.version}-setuptools \
                         port:swig-python \

    depends_lib-append   port:fann

    livecheck.type       none
```

Is `fann` needed for the build process? I presumed that it should be used as a library.
I think replacing `swig` with `python-swig` should help fix the error, I tested it [here](https://travis-ci.com/github/harens/macports-ports) and it seems to work.